test/ulid.spec.js

```diff
@@ -190,4 +192,22 @@ describe("ulid", function() {
             expect(ulid(1469918176385).substring(0, 10)).to.equal("01ARYZ6S41");
         });
     });
+
+    describe("isValid", function() {
+        it("should return true for valid ULIDs", function() {
+            expect(isValid("01ARYZ6S41TSV4RRFFQ69G5FAV")).to.be.true;
+        });
+
+        it("should return false for invalid ULIDs (wrong length)", function() {
+            expect(isValid("01ARYZ6S41TSV4RRFFQ69G5FA")).to.be.false;
+        });
+
+        it("should return false for invalid ULIDs (wrong characters)", function() {
+            expect(isValid("01ARYZ6S41TSV4RRFFQ69G5FA!")).to.be.false;
+        });
+
+        it("should return false for invalid ULIDs (wrong type)", function() {
+            expect(isValid(123)).to.be.false;
+        });
```

Comment:
Please add a test here for a lowercase ULID.

Reply:
lowercase test added